client/ayon_openrv/api/review.py

```diff
@@ -4,14 +4,14 @@
 import rv
 
 
-def get_path_annotated_frame(frame=None, asset=None, asset_folder=None):
+def get_path_annotated_frame(frame=None, folder_name=None, folder_path=None):
     """Get path for annotations
     """
     # TODO: This should be less hardcoded
     filename = os.path.normpath(
         "{}/pyblish/exports/annotated_frames/annotate_{}_{}.jpg".format(
```

Comment:
Might want to also get rid of the `pyblish` here and replace it with `ayon`?
```suggestion
        "{}/ayon/exports/annotated_frames/annotate_{}_{}.jpg".format(
```